solidity/ecdsa/deploy/10_deploy_proxy_admin_with_deputy.ts

```diff
@@ -0,0 +1,27 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
```

Comment:
I think we could theoretically reuse `ProxyAdminWithDeputy` we will use for the staking contract, because there is no state in the `ProxyAdmin` specific to proxy or implementation contract. On the other hand, it feels cleaner to have a separate admin per proxy contract. In case we deploy another application contract with another type of proxy, we may need a separate `ProxyAdminWithDeputy` anyway. See https://github.com/threshold-network/solidity-contracts/pull/21.

cc @cygnusv

Reply:
5bfc5285996d172a5bb03f1cc2beb91eda60f163